### PR TITLE
Feature/gh 615

### PR DESCRIPTION
**Pull request by mikelMHybris**

### CODE REVIEW

**Comment by developpeurweb** [comment]
Fixes GH-615

File: projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss [hunk 1]
```diff
@@ -1,109 +1,71 @@
...
+}
 
-$cx-search-grid-price-old-color: 'secondary' !default;
-$cx-search-grid-price-old-trbl-margin: 0 0 0 0 !default;
-$cx-search-grid-price-new-color: 'primary' !default;
-$cx-search-grid-price-new-trbl-margin: 0 0 25px 5px !default;
+.cx-product-image {
+  width: var(--cx-width, 100%);
+  padding: var(--cx-padding, 20px);
+  display: var(--cx-display, block);
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss]: The break point seems to be missing.

File: projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss [hunk 2]
```diff
@@ -1,109 +1,71 @@
...
-.cx-product {
-  &-search-grid__image--container {
-    display: block;
-    text-align: $cx-search-grid-image-align;
+.cx-product-name {
+  @include type('5');
+  text-align: var(--cx-text-align, center);
+  display: var(--cx-display, block);
+  margin: var(--cx-margin, 0 0 25px 0);
+  color: var(--cx-g-color-dark);
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss]: This should be `color: var(--cx-color, --cx-g-color-dark);`

File: projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss [hunk 3]
```diff
@@ -1,109 +1,71 @@
...
-    padding: $cx-search-grid-image-trbl-padding;
-    display: block;
-
-    @include media-breakpoint-down(sm) {
-      padding-top: $cx-search-grid-image-sm-padding-top;
-    }
+  &:after {
+    content: '';
+    background: var(--cx-background, none repeat scroll 0% 0%);
+    background-color: var(--cx-g-color-inverse);
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss]: This should be`background-color: var(--cx-background-color, --cx-g-color-inverse);`

File: projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss [hunk 4]
```diff
@@ -1,109 +1,71 @@
...
-      height: $cx-search-grid-name-after-height;
-      width: $cx-search-grid-name-after-width;
-      z-index: 1;
-    }
-
-    &:hover {
-      @include var-color('color', $cx-search-grid-name-hover-color);
-    }
+  &:hover {
+    color: var(--cx-g-color-primary);
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss]: This should be `color: var(--cx-color, --cx-g-color-primary);`

File: projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss [hunk 5]
```diff
@@ -1,109 +1,71 @@
...
+  display: var(--cx-display, inline-block);
+  margin: var(--cx-margin, 0 0 25px 0);
 
-    &--old {
-      @include var-color('color', $cx-search-grid-price-old-color);
-      text-decoration: line-through;
-      margin: $cx-search-grid-price-old-trbl-margin;
-    }
+  .old {
+    color: var(--cx-g-secondary);
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss]: This should be `color: var(--cx-color, --cx-g-color-secondary);`

File: projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss [hunk 6]
```diff
@@ -1,109 +1,71 @@
...
+    margin: var(--cx-margin, 0);
+  }
 
-    &--new {
-      margin: $cx-search-grid-price-new-trbl-margin;
-      @include var-color('color', $cx-search-grid-price-new-color);
-    }
+  .new {
+    margin: var(--cx-margin, 0 0 25px 5px);
+    color: var(--cx-g-primary);
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss]: This should be color: var(--cx-color, --cx-g-color-primary);

**Pull request merged**